bin/improver-tests

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+#------------------------------------------------------------------------------
+# (C) British Crown Copyright 2017 Met Office.
+#------------------------------------------------------------------------------
+# NAME
+#    improver tests - Run IMPROVER self-tests
+#
+# SYNOPSIS
+#    improver tests
+#
+# DESCRIPTION
+#    Launch all IMPROVER self-tests.
+#------------------------------------------------------------------------------
+
+set -eu
+
+function echo_ok {
+    echo -e "\033[1;32m[OK]\033[0m $1"
+}
+
```

Comment:
I think this routine needs to have a descriptive help message added so calling it with "improver help tests" tells the user what it is actually doing, rather than just running the scripts. That would make its behaviour more consistent.

Something like the following:
```
if (($# != 0)); then
    if [[ $1 == help ]] || [[ $1 == --help ]]; then
        echo 'here is some help'
        exit 0
    else
        echo 'Unknown option sent to test'
    fi
fi
```